JobRunner/callback_server.py

```diff
@@ -85,7 +85,3 @@ def start_callback_server(ip, port, out_queue, in_queue, token):
     }
     app.config.update(conf)
     app.run(host=ip, port=port, debug=False, access_log=False)
-
-
-if __name__ == "__main__":
```

Comment:
It could be useful to be able to run the callback as a script.

Reply:
There is other stuff needed.  So we would need to think through the execution model.